diff --git a/src/storage/schema_sqlite.rs b/chirpstack/src/storage/schema_sqlite.rs
index 739ff7c0..14aff522 100644
--- a/src/storage/schema_sqlite.rs
+++ b/src/storage/schema_sqlite.rs
@@ -1,50 +1,50 @@
 // @generated automatically by Diesel CLI.
 
 diesel::table! {
     api_key (id) {
         id -> Text,
-        created_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
         name -> Text,
         is_admin -> Bool,
         tenant_id -> Nullable<Text>,
     }
 }
 
 diesel::table! {
     application (id) {
         id -> Text,
         tenant_id -> Text,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
         name -> Text,
         description -> Text,
         mqtt_tls_cert -> Nullable<Binary>,
         tags -> Text,
     }
 }
 
 diesel::table! {
     application_integration (application_id, kind) {
         application_id -> Text,
         kind -> Text,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
         configuration -> Text,
     }
 }
 
 diesel::table! {
     device (dev_eui) {
         dev_eui -> Binary,
         application_id -> Text,
         device_profile_id -> Text,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
-        last_seen_at -> Nullable<Timestamp>,
-        scheduler_run_after -> Nullable<Timestamp>,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
+        last_seen_at -> Nullable<TimestamptzSqlite>,
+        scheduler_run_after -> Nullable<TimestamptzSqlite>,
         name -> Text,
         description -> Text,
         external_power_source -> Bool,
         battery_level -> Nullable<Double>,
         margin -> Nullable<Integer>,
         dr -> Nullable<SmallInt>,
@@ -63,27 +63,27 @@ diesel::table! {
     }
 }
 
 diesel::table! {
     device_keys (dev_eui) {
         dev_eui -> Binary,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
         nwk_key -> Binary,
         app_key -> Binary,
         dev_nonces -> Text,
         join_nonce -> Integer,
     }
 }
 
 diesel::table! {
     device_profile (id) {
         id -> Text,
         tenant_id -> Text,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
         name -> Text,
         region -> Text,
         mac_version -> Text,
         reg_params_revision -> Text,
         adr_algorithm_id -> Text,
         payload_codec_runtime -> Text,
@@ -135,14 +135,14 @@ diesel::table! {
     }
 }
 
 diesel::table! {
     device_profile_template (id) {
         id -> Text,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
         name -> Text,
         description -> Text,
         vendor -> Text,
         firmware -> Text,
         region -> Text,
         mac_version -> Text,
@@ -172,30 +172,30 @@ diesel::table! {
 }
 
 diesel::table! {
     device_queue_item (id) {
         id -> Text,
         dev_eui -> Binary,
-        created_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
         f_port -> SmallInt,
         confirmed -> Bool,
         data -> Binary,
         is_pending -> Bool,
         f_cnt_down -> Nullable<BigInt>,
-        timeout_after -> Nullable<Timestamp>,
+        timeout_after -> Nullable<TimestamptzSqlite>,
         is_encrypted -> Bool,
     }
 }
 
 diesel::table! {
     gateway (gateway_id) {
         gateway_id -> Binary,
         tenant_id -> Text,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
-        last_seen_at -> Nullable<Timestamp>,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
+        last_seen_at -> Nullable<TimestamptzSqlite>,
         name -> Text,
         description -> Text,
         latitude -> Double,
         longitude -> Double,
         altitude -> Float,
         stats_interval_secs -> Integer,
@@ -206,14 +206,14 @@ diesel::table! {
 }
 
 diesel::table! {
     multicast_group (id) {
         id -> Text,
         application_id -> Text,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
         name -> Text,
         region -> Text,
         mc_addr -> Binary,
         mc_nwk_s_key -> Binary,
         mc_app_s_key -> Binary,
         f_cnt -> BigInt,
@@ -226,29 +226,29 @@ diesel::table! {
 }
 
 diesel::table! {
     multicast_group_device (multicast_group_id, dev_eui) {
         multicast_group_id -> Text,
         dev_eui -> Binary,
-        created_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
     }
 }
 
 diesel::table! {
     multicast_group_gateway (multicast_group_id, gateway_id) {
         multicast_group_id -> Text,
         gateway_id -> Binary,
-        created_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
     }
 }
 
 diesel::table! {
     multicast_group_queue_item (id) {
         id -> Text,
-        created_at -> Timestamp,
-        scheduler_run_after -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        scheduler_run_after -> TimestamptzSqlite,
         multicast_group_id -> Text,
         gateway_id -> Binary,
         f_cnt -> BigInt,
         f_port -> SmallInt,
         data -> Binary,
         emit_at_time_since_gps_epoch -> Nullable<BigInt>,
@@ -256,21 +256,21 @@ diesel::table! {
 }
 
 diesel::table! {
     relay_device (relay_dev_eui, dev_eui) {
         relay_dev_eui -> Binary,
         dev_eui -> Binary,
-        created_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
     }
 }
 
 diesel::table! {
     tenant (id) {
         id -> Text,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
         name -> Text,
         description -> Text,
         can_have_gateways -> Bool,
         max_device_count -> Integer,
         max_gateway_count -> Integer,
         private_gateways_up -> Bool,
@@ -280,26 +280,26 @@ diesel::table! {
 }
 
 diesel::table! {
     tenant_user (tenant_id, user_id) {
         tenant_id -> Text,
         user_id -> Text,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
         is_admin -> Bool,
         is_device_admin -> Bool,
         is_gateway_admin -> Bool,
     }
 }
 
 diesel::table! {
     user (id) {
         id -> Text,
         external_id -> Nullable<Text>,
-        created_at -> Timestamp,
-        updated_at -> Timestamp,
+        created_at -> TimestamptzSqlite,
+        updated_at -> TimestamptzSqlite,
         is_admin -> Bool,
         is_active -> Bool,
         email -> Text,
         email_verified -> Bool,
         password_hash -> Text,
         note -> Text,
